build/android/src/main/java/net.minetest.minetest/MtNativeActivity.java

```diff
@@ -19,6 +19,10 @@ public void onCreate(Bundle savedInstanceState) {
 	public void onDestroy() {
 		super.onDestroy();
 	}
+	
+	@Override
+	public void onBackPressed() {
```

Comment:
??

You want the bug to be fixed, its very important to you, no?

I have tried really hard to improve your PR and give you suggestions but you dont edit it. And now I do it myself (and creating that patch file wasn't as easy as it looks), and you say no? Do you want it to be fixed or not? I credit you in the commit description, if its that what upsets you.